components/MegaMenu.vue

```diff
@@ -6,16 +6,39 @@
       :title="category.content.name"
       :link="localePath(getCatLink(category))"
     >
-      <SfList>
-        <SfListItem v-for="child in category.childrenCategories" :key="child.id">
-          <SfMenuItem :label="$t(child.content.name)" :link="localePath(getCatLink(child))" />
-        </SfListItem>
-      </SfList>
+      <div class="sf-mega-menu-option">
+        <div class="flex-grow-3">
+          <SfList class="outer-list">
+            <SfListItem v-for="child in category.childrenCategories" :key="child.id">
+              <SfMenuItem class="sf-title" :label="child.content.name" />
+              <SfMenuItem :link="localePath(getCatLink(child))" :label="$t('ShopAll')" />
+              <SfList>
+                <SfListItem v-for="children in child.childrenCategories" :key="children.id">
```

Comment:
rename `children` to `grandChild`. 
`children` is plural and sounds like a list

Reply:
done